src/js/routes/services.js

```diff
@@ -1,37 +1,69 @@
 import Router from 'react-router';
-let Route = Router.Route;
+let {Route, Redirect} = Router;
 
+import DeploymentsTab from '../pages/services/DeploymentsTab';
 import ServiceOverlay from '../components/ServiceOverlay';
 import ServicesPage from '../pages/ServicesPage';
+import ServicesTab from '../pages/services/ServicesTab';
 
-let servicesRoutes = {
+let services = {
```

Comment:
I see where you're coming from. Not sure we need to go as far as renaming the modules (since they are nested within the routes directory, thus providing context) but it would be nice if we at least imported like `import servicesRoutes from './routes/services';`. But I'd suggest that is out of scope here.